src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdk.diff

```diff
@@ -63,13 +45,310 @@ index ------------
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/analyzers/
 @@ ------------ @@
+ ./packs/Microsoft.NETCore.App.Ref/x.y.z/ref/
+ ./packs/Microsoft.NETCore.App.Ref/x.y.z/ref/netx.y/
+ ./packs/Microsoft.NETCore.App.Ref/x.y.z/ref/netx.y/Microsoft.CSharp.dll
+-./packs/Microsoft.NETCore.App.Ref/x.y.z/ref/netx.y/Microsoft.CSharp.xml
```

Comment:
Missing ref packs issue has been fixed but is waiting to flow in - https://github.com/dotnet/source-build/issues/2877